msgid ""
msgstr ""
"Project-Id-Version: pylav\n"
"POT-Creation-Date: 2023-01-31 23:27+0000\n"
"PO-Revision-Date: 2023-02-02 20:05\n"
"Last-Translator: \n"
"Language-Team: Croatian\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 3.4.2\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
"X-Crowdin-Project: pylav\n"
"X-Crowdin-Project-ID: 535244\n"
"X-Crowdin-Language: hr\n"
"X-Crowdin-File: /lib/pylav/extension/bundled_node/locales/messages.pot\n"
"X-Crowdin-File-ID: 3385\n"
"Language: hr_HR\n"

#: pylav/extension/bundled_node/manager.py:227
msgid "You are attempting to run the managed Lavalink node on an unsupported machine architecture"
msgstr "Pokušavate pokrenuti upravljani čvor Lavalink na nepodržanoj arhitekturi stroja"

#: pylav/extension/bundled_node/manager.py:452
msgid "The output of `{java_exc} -version` was unexpected\n"
"{version_info}"
msgstr "Izlaz od '{java_exc} -verzija' je neočekivano\n"
"{version_info}"

#: pylav/extension/bundled_node/manager.py:472
msgid "Port {port} already in use. Managed Lavalink startup aborted"
msgstr "Luka {port} već se koristi. Prekinuto upravljano pokretanje Lavalinka"

#: pylav/extension/bundled_node/manager.py:477
msgid "Lavalink failed to start: {line}"
msgstr "Lavalink se nije uspio pokrenuti: {line}"

